README.md

```diff
@@ -28,14 +28,24 @@ Be prepared to demonstrate your understanding of this week's concepts by answeri
 
 1. What is the DOM?
 
+The DOM or Document Object Model is used to manipulate an HTML document by changing, adding or removing content.
```

Comment:
The DOM is an object representation of the html. The DOM can me manipulated using javascript